lib/ares/version.rb

```diff
@@ -1,3 +1,3 @@
 module Ares
-  VERSION = "0.1.0"
+  VERSION = "0.2.0"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.